ddl/db_test.go

```diff
@@ -4487,6 +4487,10 @@ func (s *testDBSuite4) TestIfExists(c *C) {
 	s.mustExec(tk, c, "alter table t2 drop partition if exists p1")
 	c.Assert(tk.Se.GetSessionVars().StmtCtx.WarningCount(), Equals, uint16(1))
 	tk.MustQuery("show warnings").Check(testutil.RowsWithSep("|", "Note|1507|Error in list of partitions to p1"))
+
+	// DROP INDEXES
+	s.mustExec(tk, c, "alter table t1 drop index if exists idxes_a,drop index if exists idxes_b")
```

Comment:
Please add some tests, mixed with "if exist" and without "if exist", including successful and failed execution.

Reply:
I have upgraded my test cases.